### ISSUE TITLE
specifying/creating device (/dev/xxx) which  signalk-to-nmea0183 writes to

### ISSUE THREAD

**Issue by kaililleby** [SignalK/signalk-to-nmea0183#11]
Greeting from Norway :) 
Im playing with openplotter on a boat with a nmea2000 bus, signal k receives data (through actisense ngt-1), but I haven't managed to forward those data to opencpn. I thought I could use signalk-to-nmea0183 for this task, writing to a serial device, and setup opencpn to read from this. Is this the intended way? If so - do I create a new serial port on the rpi, and set "device" in the signalk config to this? 
Best regards
Kai

**Comment by tkurki** [comment]
I am not an OpenCpn user, but I don’t think a serial out is the way to go. By default signalk server is serving nmea0183 over tcp on port 10110, but I assume OpenPlotter has that port occupied with something else. Maybe you can configure sk server to have nmea0183@tcp on some other port and configure OpenCpn to connect to that? 

See https://github.com/SignalK/signalk-server-node#environment-variables

Ping @sailoog, we have discussed n2k -> sk server -> 0183 -> OpenCpn previously.

**Comment by kaililleby** [comment]
great! do you have a config example for serving nmea0183 over tcp?

**Comment by tkurki** [comment]
Default configuration for sk server is that nmea0183 is served over tcp on port 10110, but like I said I don't know the details of OpenPlotter SK configuration, there might be something else on that port. Or it might work out of the box.

You can try things out on:
- install SK Node server
- start with `bin/n2k-from-file`
- enable the conversion plugin & activate some conversion like RMC
- `telnet localhost 10110`

**Comment by sailoog** [comment]
No, we are not using tcp 10110 in openplotter. We left free this so that signal k can use it. We use udp 10110 for kplex input and tcp 10109 to send NMEA 0183 to opencpn from kplex.

The best option for you would be n2k -> sk server -> signalk-to-nmea0183 -> tcp 10110 input in opencpn
